statsmodels/base/_parameter_inference.py

```diff
@@ -5,17 +5,11 @@
 @author: josef
 """
 
-import time
 import numpy as np
 from scipy import stats
 
-from statsmodels.regression.linear_model import OLS
-from statsmodels.genmod.generalized_linear_model import GLM
-from statsmodels.genmod import families
-import statsmodels.stats._diagnostic_other as diao
 
-
-# this is a copy from stats._diagnostic_other
+# this is a copy from stats._diagnostic_other to avoid circular imports
 def _lm_robust(score, constraint_matrix, score_deriv_inv, cov_score,
```

Comment:
This copy should be removed and it should be imported right before first use, with a note explaining why it is imported there.

Reply:
Or just don’t remove the import of disgnostic_other at the top of the file